graphql_query_examples/graphql_query_examples.md

```diff
@@ -1200,6 +1200,56 @@ query get_results_for_list_of_checks {
 ```
 </details>
 
+### 🔎 services > query for all services with openapi docs
+
+If an OpenAPI doc is found for a service, the `preferredApiDocumentSource` 
+field will be set. Otherwise `preferredApiDocumentSource` will be null.
+
+```graphql
+query all_services_with_openapi_docs($endCursor: String) {
+  account {
+    services(after: $endCursor) {
+      nodes {
+        name
+        preferredApiDocument {
+          id
+          content
+        }
+      }
+      pageInfo {
+        endCursor
+        hasNextPage
+      }
+    }
+  }
+}
+```
+
+<details>
+  <summary>Usage with opslevel-cli expand to show</summary>
+The opslevel-cli command below will return a list of services by name that have `preferredApiDocumentSource` set.
+
+```bash
+opslevel graphql --paginate -a=".account.services.nodes[]" -q='query all_services_with_openapi_docs($endCursor: String) {
+  account {
+    services(after: $endCursor) {
+      nodes {
+        name
+        preferredApiDocument {
+          id
+          content
+        }
+      }
+      pageInfo {
+        endCursor
+        hasNextPage
+      }
+    }
+  }
+}' | jq '.[] | select(.preferredApiDocument.id != null and .preferredApiDocument.content != null) | .name'
```

Comment:
this works but it's sufficient and simpler to just check if `content` is null. It will be confusing to users why we're telling them to check for both

Reply:
@Farjaad changes made!